config/config_test.go

```diff
@@ -46,6 +47,8 @@ func (s *testConfigSuite) TestConfig(c *C) {
 	// Test that the value will be overwritten by the config file.
 	c.Assert(conf.Performance.RetryLimit, Equals, 10)
 
+	c.Assert(conf.TiKVClient.CommitTimeout, Equals, "41s")
```

Comment:
Seems you  set 10s above.

Reply:
It will be set `41s` after this
```
configFile := path.Join(path.Dir(filename), "config.toml.example")
```
In example file, it is `41s`